modules/virt-configuring-live-migration-heavy.adoc

```diff
@@ -0,0 +1,63 @@
+
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-configuring-live-migration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-live-migration-heavy_{context}"]
+= Configure live migration for heavy workloads
+
+When migrating a VM running a heavy workload (for example, database processing) with higher memory dirty rates, you need a higher bandwidth to complete the migration.
+
+If the dirty rate is too high, this prevents the migration from one node to another from converging and results in data loss. To prevent this, enable post copy mode.
+
+Post copy mode triggers if the initial pre-copy phase does not complete within the configured timeout parameter. During post copy, the VM CPUs pause on the source host while transfering the minimum required memory pages. Then the VM CPUs activate on the destination host, and the remaining memory pages transfer into the destination node at runtime.
+
+Configure live migration for heavy workloads by updating the `HyperConverged` custom resource (CR), which is located in the `{CNVNamespace}` namespace.
+
+.Procedure
+
+* Edit the `HyperConverged` CR and add the necessary parameters for migrating heavy workloads:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n {CNVNamespace}
+----
++
+.Example configuration file
+[source,yaml,subs="attributes+"]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: {CNVNamespace}
+spec:
+  liveMigrationConfig:
+    bandwidthPerMigration: 64Mi <1>
+    completionTimeoutPerGiB: 800 <2>
+    parallelMigrationsPerCluster: 5 <3>
+    parallelOutboundMigrationsPerNode: 1 <4>
+    progressTimeout: 150 <5>
+    allowPostCopy: true <6>
+----
+<1> Bandwidth limit of each migration, where the value is the quantity of bytes per second. For example, a value of `2048Mi` means 2048 MiB/s. Default: `0`, which is unlimited.
+//What should this be set to for heavy workloads?
```

Comment:
Question 1

Reply:
To me this is more about external infrastructure constraints. As far as migrations are concerned, faster is better, so 0 is always best.
If this is set to a non-zero value, the lower the value is the higher the chances are that migrations will switch to post-copy (or fail if post-copy is not allowed).